### PR TITLE
Oslc4j plugin and Feature

### PR DESCRIPTION
**Pull request by yashkhatri**
Created a OSLC4J  plugin project and a Feature. Included the feature in the update site. Also fixed the Emf2Rdf and include it in tools feature. (Summary: mvn clean package should work fine.)

### CODE REVIEW

**Comment by yashkhatri** [comment]
@jadelkhoury @berezovskyi I have created a oslc4j-plugin project where I have included the needed oslc core projects and exported the needed packages. (Needed for export project). mvn clean package works fine on my machine (ignored acceleo compile goal). Created an update site and installed locally and verified. However, please verify on your machines too.
 
Before merging the branch to master, I would like to refactor the export project too. Or may be we can remove that from this branch and merge the plugin and feature.

We can ignore all other attempts to make the oslc4j plugin/bundle, export.

File: org.eclipse.lyo.oslc4j.feature/feature.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      Copyright (c) 2018 Yash Khatri and others.
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License
+v1.0 and Eclipse Distribution License v. 1.0 which accompanies
+this distribution.
+The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+and the Eclipse Distribution License is available at http://www.eclipse.org/org/documents/edl-v10.php.
+   </copyright>
+
+   <license url="http://www.example.com/license">
```
Comment (yashkhatri) [inline org.eclipse.lyo.oslc4j.feature/feature.xml]: Should be "https://eclipse.org/legal/epl/notice.php"

File: org.eclipse.lyo.oslc4j.feature/feature.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<feature
+      id="org.eclipse.lyo.oslc4j.feature"
+      label="Oslc4j Core Plugin"
+      version="2.4.0.qualifier">
+
+   <description url="http://www.example.com/description">
+      OSLC4J feature contains the plugin of all OSLC4J Core projects.s
+   </description>
+
+   <copyright url="http://www.example.com/copyright">
```
Comment (yashkhatri) [inline org.eclipse.lyo.oslc4j.feature/feature.xml]: Should be "https://www.eclipse.org/lyo/"

File: org.eclipse.lyo.oslc4j.feature/feature.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.eclipse.lyo.oslc4j.feature"
+      label="Oslc4j Core Plugin"
+      version="2.4.0.qualifier">
+
+   <description url="http://www.example.com/description">
```
Comment (yashkhatri) [inline org.eclipse.lyo.oslc4j.feature/feature.xml]: Should be "https://www.eclipse.org/lyo/"

File: org.eclipse.lyo.oslc4j.plugin/target/local-artifacts.properties [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#Wed Oct 24 01:26:16 CEST 2018
```
Comment (yashkhatri) [inline org.eclipse.lyo.oslc4j.plugin/target/local-artifacts.properties]: target folder should be removed.

**Comment by berezovskyi** [comment]
1) This patch just manually includes all JARs. Seems like a hack?
2) Are you targeting this patch for the Lyo 2.4.0 release? I thought we decided to drop it and get the validation fixed up instead in the few days we have left.

**Comment by yashkhatri** [comment]
[pom.txt](https://github.com/eclipse/lyo.designer/files/2510170/pom.txt)

Here's a pom file of oslc4j-plugin project that I have use for copying the dependencies in the lib folder. (uploading in .txt format)

**Comment by jadelkhoury** [comment]
Let's wait till after the release.
@berezovskyi is this good enough, or still a hack?

**Comment by berezovskyi** [comment]
Still a hack :)

**Pull request closed without merging**